### PR TITLE
Projectile Dismemberment and Wizard spellblade

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
add: Added support for projectiles which can dismember
add: Added a new wizard sword, the spellblade, which fires dismembering projectiles. Sprites from Lexorion.
add: Plasmacutter blasts can now take limbs off, but they still do pitiful damage in a pressurized environment.
/:cl:

Added dismembering projectiles.

Added a new wizard weapon that takes advantage of this (possibly a bad idea!) Sprites from Lexorion.

Added said wizard weapon as a bubblegum drop (possibly a worse idea!)

Added a chaplain weapon skin variant (of course) with no special properties

Plasmacutter blasts can now dismember.

Used some of the new @Cheridan sounds for chaplain weapons.

### CODE REVIEW

**Comment by GunHog** [comment]
Isaac Clarke approved!

**Comment by Lexorion** [comment]
spellblade

**Comment by GunHog** [comment]
I had the same thought, PKP. Terrablade and swords like it.

**Comment by KorPhaeron** [comment]
@PKPenguin321 

It'd be really tricky since you'd have to do it per pellet, and you could end up in weird scenarios where you get hit by one pellet from down the hallway on an already wounded limb and it just comes flying off

**Comment by optimumtact** [comment]
The only weapon in our arsenal that might dismember someone would be the sniper rifle.

**Comment by KorPhaeron** [comment]
@optimumtact 

Don't forget the head of security gun

**Comment by optimumtact** [comment]
Don't even start

File: code/modules/mob/living/carbon/human/human_defense.dm [hunk 1]
```diff
@@ -60,7 +60,16 @@
 		if(check_shields(P.damage, "the [P.name]", P, PROJECTILE_ATTACK, P.armour_penetration))
 			P.on_hit(src, 100, def_zone)
 			return 2
-	return (..(P , def_zone))
+
+	var/armor = run_armor_check(def_zone, P.flag, "","",P.armour_penetration)
+
+	if(!P.nodamage)
+		apply_damage(P.damage, P.damage_type, def_zone, armor)
```
Comment (phil235) [inline code/modules/mob/living/carbon/human/human_defense.dm]: why is this here.....

Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/human_defense.dm]: because I'm dumb and mob code is everywhere and confusing

Comment (KorPhaeron) [inline code/modules/mob/living/carbon/human/human_defense.dm]: I'll find a better place to put the dismemberment thing after dinner

**Comment by TechnoAlchemisto** [comment]
this is the tightest shit

**Comment by Kromgar** [comment]
Bring back multiverse sword

**Comment by Niknakflak** [comment]
Damn that sprite is sexy
